packages/libs/eda/src/lib/map/analysis/MarkerConfiguration/BarPlotConfigurationMenu.tsx

```diff
@@ -0,0 +1,99 @@
+import { H6 } from '@veupathdb/coreui';
+import {
+  InputVariables,
+  Props as InputVariablesProps,
+} from '../../../core/components/visualizations/InputVariables';
+import RadioButtonGroup from '@veupathdb/components/lib/components/widgets/RadioButtonGroup';
+import { VariableDescriptor } from '../../../core/types/variable';
+import { useState } from 'react';
+import { VariablesByInputName } from '../../../core/utils/data-element-constraints';
+
+interface MarkerConfiguration<T extends string> {
+  type: T;
+}
+
+export interface BarPlotMarkerConfiguration
+  extends MarkerConfiguration<'barplot'> {
+  selectedVariable: VariableDescriptor;
+  selectedPlotMode: 'count' | 'proportion';
+}
+
+interface Props
+  extends Omit<
+    InputVariablesProps,
+    'onChange' | 'selectedVariables' | 'selectedPlotMode' | 'onPlotSelected'
+  > {
+  selectedPlotMode: string;
+  onChange: (configuration: BarPlotMarkerConfiguration) => void;
+  configuration: BarPlotMarkerConfiguration;
+}
+
+export function BarPlotConfigurationMenu({
+  entities,
+  onChange,
+  starredVariables,
+  toggleStarredVariable,
+  configuration,
+}: Props) {
+  function handleInputVariablesOnChange(selection: VariablesByInputName) {
+    if (!selection.overlay) {
+      console.error(
+        `Expected overlay to defined but got ${typeof selection.overlay}`
+      );
+      return;
```

Comment:
<img width="727" alt="image" src="https://user-images.githubusercontent.com/24446573/235537346-db34a16e-950b-47dd-bb06-8734d5e7caad.png">

☝️ I imagine that we want to update this component to take a prop to hide the clear button. If not, I suppose we could select the study's default overlay variable.